.github/config/evergreen-arm-hardfp-rdk.json

```diff
@@ -0,0 +1,40 @@
+{
+  "docker_service": "build-rdk",
```

Comment:
This should stay `build-raspi` until there is a `build-rdk` docker service. See defined docker services in `docker-compose.yml`.